### PR TITLE
Fonts japanese translation

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586

Please review.

### CODE REVIEW

File: src/content/ja/2019/fonts.md [hunk 1]
```diff
@@ -0,0 +1,651 @@
...
+
+最初の、そして最も顕著な問題は、パフォーマンスです。[パフォーマンス](./performance)に特化した章がありますが、ここではフォント固有のパフォーマンスの問題について少し掘り下げてみましょう。
+
+ホストされたWebフォントを使用すると、実装やメンテナンスが容易になりますが、セルフホスティングは最高のパフォーマンスを提供します。Webフォントはデフォルトで、Webフォントの読み込み中にテキストを非表示にする（[Flash of Invisible Text](https://css-tricks.com/fout-foit-foft/)、またはFOITとしても知られています）ことを考えると、Webフォントのパフォーマンスは画像のような非ブロッキング資産よりも重要になる可能性があります。
+
+### フォントは同じホストでホストされているのか、それとも別のホストでホストされているのか？
+
+サードパーティのホスティングに対するセルフホスティングの差別化は、[HTTP/2](./http2)の世界ではますます重要になってきています。同一ホストのリクエストには、ウォーターフォール内の他の同一ホストのリクエストに対して優先順位をつける可能性が高いという大きな利点があります。
+
+別のホストからウェブフォントを読み込む際のパフォーマンスコストを軽減するための推奨事項としては、`preconnect`、`dns-prefetch`、`preload` [resource-hints](./resource-hints)の使用がありますが、優先度の高いウェブフォントは、ウェブフォントのパフォーマンスへの影響を最小限に抑えるため、同一ホストからのリクエストにすべきです。これは視覚的に、非常に目立つコンテンツやページの大部分を占める本文コピーで使用されるフォントへ対して特に重要です。
```
Comment (bazzadp) [inline src/content/ja/2019/fonts.md:32]: Previously you've translates recource-hints
```suggestion
別のホストからウェブフォントを読み込む際のパフォーマンスコストを軽減するための推奨事項としては、`preconnect`、`dns-prefetch`、`preload` [リソースのヒント](./resource-hints)の使用がありますが、優先度の高いウェブフォントは、ウェブフォントのパフォーマンスへの影響を最小限に抑えるため、同一ホストからのリクエストにすべきです。これは視覚的に、非常に目立つコンテンツやページの大部分を占める本文コピーで使用されるフォントへ対して特に重要です。
```

File: src/content/ja/2019/fonts.md [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+    </tbody>
+  </table>
+  <figcaption>図2. リクエストの割合による上位20のフォントホスト。</figcaption>
+</figure>
+
+ここでのGoogle Fontsの優位性は、同時に驚くべきことであると同時に意外性のないものであった。期待していたという点では予想外でしたが、サービスの圧倒的な人気の高さには驚きました。フォントリクエストの75％というのは驚異的だ。TypeKitは一桁台の遠い2位で、Bootstrapライブラリがさらに遠い3位を占めていました。
+
+<figure>
+  <div class="big-number">29%</div>
+  <figcaption>図3. ドキュメント内にGoogle Fontsスタイルシートのリンクを含むページの割合 <code>&lt;head></code>。</figcaption>
```
Comment (bazzadp) [inline src/content/ja/2019/fonts.md:169]: Looks wrong in English. Will fix in other chapters, but let's fix Japanese here.
```suggestion
  <figcaption>図3. ドキュメント内にGoogle Fontsスタイルシートのリンクを含むページの割合 <code>&lt;head&gt;</code>。</figcaption>
```

File: src/content/ja/2019/fonts.md [hunk 3]
```diff
@@ -0,0 +1,651 @@
...
+
+ここでのGoogle Fontsの使用率の高さは非常に印象的だが、Google Fonts`<link>`要素を含むページが29％しかなかったことも注目に値する。これはいくつかのことを意味しているかもしれない。
+
+- ページがGoogle Fontsを使用する場合、彼らはGoogle Fontsを多く使用しています。それらは結局のところ、金銭的なコストなしで提供されています。おそらく、人気のあるWYSIWYGエディタで使われているのではないでしょうか？　これは非常に可能性の高い説明のように思えます。
+- あるいは、もっとありそうにない話としては、多くの人が`<link>`の代わりに`@import`を使ってGoogle Fontsを使っているということかもしれません。
+- あるいは、超ありえないシナリオにまで踏み込んでみたいのであれば、多くの人が代わりに[HTTP `Link:`ヘッダー](https://developer.mozilla.org/ja/docs/Web/HTTP)を使ってGoogle Fontsを使っているということになるかもしれません。
+
+<figure>
+  <div class="big-number">0.4%</div>
+  <figcaption>図4. ドキュメントの最初の子としてGoogle Fontsスタイルシートのリンクを含むページの割合 <code>&lt;head></code>。</figcaption>
```
Comment (bazzadp) [inline src/content/ja/2019/fonts.md:180]:
```suggestion
  <figcaption>図4. ドキュメントの最初の子としてGoogle Fontsスタイルシートのリンクを含むページの割合 <code>&lt;head&gt;</code>。</figcaption>
```

File: src/content/ja/2019/fonts.md [hunk 4]
```diff
@@ -0,0 +1,651 @@
...
+
+第3位の`octet-stream`(およびもう少し下の`plain`)は、多くのウェブサーバが不適切に設定されており、ウェブフォントファイルのリクエストで誤ったMIMEタイプを送信していることを示唆しているように見えます。
+
+もう少し深く掘り下げて、`@font-face`宣言の`src:`プロパティで使われている`format()`の値を見てみましょう。
+
+<figure>
+  <a href="/static/images/2019/fonts/fig8.png">
+    <img src="/static/images/2019/fonts/fig8.png" alt="図8. <code>@font-face</code>宣言におけるフォントフォーマットの人気度。" aria-labelledby="fig8-caption" aria-describedby="fig8-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=700778025&amp;format=interactive">
+  </a>
+  <div id="fig8-description" class="visually-hidden">Bar chart showing the popularity of formats used in font-face declarations. 69% of desktop pages' @font-face declarations specify the WOFF2 format, 11% WOFF, 10% TrueType, 8% SVG, 2% EOT, and fewer than 1% OpenType, TTF, and OTF. The distribution for mobile pages is similar.</div>
```
Comment (bazzadp) [inline src/content/ja/2019/fonts.md:364]: Missed a translation here?

File: src/content/ja/2019/fonts.md [hunk 5]
```diff
@@ -0,0 +1,651 @@
...
+
+Google Fontsを使っているなら、スニペットを更新しよう！　Google Fontsを使っていない場合は、`font-display`を使いましょう！　`font-display`についての詳細は [MDN](https://developer.mozilla.org/ja/docs/Web/CSS/@font-face/font-display) を参照してください。
+
+どのような`font-display`値が人気あるのか見てみましょう。
+
+<figure>
+  <a href="/static/images/2019/fonts/fig11.png">
+    <img src="/static/images/2019/fonts/fig11.png" alt="図11. `font-display'の値の使用法。" aria-labelledby="fig11-caption" aria-describedby="fig11-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=1988783738&amp;format=interactive">
+  </a>
+  <div id="fig11-description" class="visually-hidden">Bar chart showing the usage of the font-display style. 2.6% of mobile pages set this style to "swap", 1.5% to "auto", 0.7% to "block", 0.4% to "fallback", 0.2% to optional, and 0.1% to "swap" enclosed in quotes, which is invalid. The desktop distribution is similar except "swap" usage is lower by 0.4 percentage points and "auto" usage is higher by 0.1 percentage points.</div>
```
Comment (bazzadp) [inline src/content/ja/2019/fonts.md:535]: Missed a translation

File: src/content/ja/2019/fonts.md (hunk 1 above)
Comment (ksakae1216) [reply src/content/ja/2019/fonts.md:32]: Thanks, I fixed.

**Comment by bazzadp** [comment]
Also can we get the featured_chapters.html updated for this chapter and added to this pull request if possible?

File: src/content/ja/2019/fonts.md (hunk 3 above)
Comment (ksakae1216) [reply src/content/ja/2019/fonts.md:180]: I fixed.

File: src/content/ja/2019/fonts.md (hunk 4 above)
Comment (ksakae1216) [reply src/content/ja/2019/fonts.md:364]: Yes, missed.
I fixed.

File: src/content/ja/2019/fonts.md (hunk 5 above)
Comment (ksakae1216) [reply src/content/ja/2019/fonts.md:535]: Thanks, I fixed.

**Comment by ksakae1216** [comment]
@bazzadp Please tell me.

Now, git push error, Is it bad for my environment?
↓↓↓
> git push origin fonts-japanese-translation:fonts-japanese-translation
remote: Permission to HTTPArchive/almanac.httparchive.org.git denied to ksakae1216.
fatal: unable to access 'https://github.com/HTTPArchive/almanac.httparchive.org.git/': The requested URL returned error: 403

**Comment by bazzadp** [comment]
@rviscomi was changing permissions earlier so suspect it's that :-(

Can you email me the file(s) and I'll add it to this branch for now?

**Comment by ksakae1216** [comment]
@bazzadp
fix files.

2020年4月25日(土) 17:20 Barry Pollard <notifications@github.com>:

**Comment by bazzadp** [comment]
Don't see files attached? Not many so do you just want to put them in as comments?

And can you clone the repo and then open a PR from that clone to here? Might have to do that for future requests anyway if we do keep it locked down.

**Comment by ksakae1216** [comment]
Try attache.

[fonts-japanese-translation.zip](https://github.com/HTTPArchive/almanac.httparchive.org/files/4532641/fonts-japanese-translation.zip)

**Comment by ksakae1216** [comment]
@bazzadp Please confirm.

**Comment by bazzadp** [comment]
Looks good. Just piushed up now and checking them now...

File: src/content/ja/2019/fonts.md (hunk 2 above)
Comment (bazzadp) [reply src/content/ja/2019/fonts.md]: Fixed this!

**Comment by bazzadp** [comment]
OK I think its good to merge. You happy or any other changes you want to make @ksakae1216 ?

**Comment by ksakae1216** [comment]
Thanks, No change.
I'm happy.

**Comment by bazzadp** [comment]
Cool. Thanks again!

You OK to move to your own cloned repo, rather than working in this main repo?

**Comment by ksakae1216** [comment]
Sorry, I don't understand.

Should I clone almanac.httparchive.org repository again?

**Comment by bazzadp** [comment]
You have been working in the main Almanac repo: https://github.com/HTTPArchive/almanac.httparchive.org by creating branches in there.

You can also work in your own clone: https://github.com/ksakae1216/almanac.httparchive.org (note this doesn't exist yet) and then request merges from that clone into the main repo. To be honest that's normally the way you would contribute to open source projects so is probably the way we should have set it up from the beginning rather than allowing everyone to create feature branches in this main repo.

**Comment by ksakae1216** [comment]
I see.
Try tomorrow, Thanks!!

**Comment by bazzadp** [comment]
Only thing with that, is remember to rebase up to the original repo frequently so your clone gets any changes! Bit more tricky than working in the main repo but fine once you get used to it.

**Comment by ksakae1216** [comment]
Oh, rebase.
Thanks, I see

**Pull request merged**